### DISCUSSION TITLE
Has open search in new tab stopped working?

### DISCUSSION THREAD

**Discussion by txtyash** [gdh1995/vimium-c#498] [Q&A] [answered]
It happened suddenly. I'm able to use open copied url in new tab, open link in new tab, etc. just fine but unable to use open the vomnibar search in newtab.

I'm using google chrome on linux.

**Comment by gdh1995** [comment]
What did you mean? If you press `O`, will Vomnibar show? If it shows, what will it do if you type some words and press `<enter>` ?

**Comment by txtyash** [comment, marked as answer]
Oh my bad! My keyboard was broken. It was sending Shift+Enter everywhere instead of Enter. I don't know why but now I've replaced it.
